Create access-comp-data

cockerilltim opened the pull request:
New section that will go into DesignSafe Essentials. This captures the Initial Onboarding and Requesting Allocations that were added after the TAPISv3 transition later in December, and I'm finally getting it into the documentation.

Updating the Nav to include new section in DesignSafe Essentials.[^1]

[^1]: The code and sentence was migrated by @wesleyboar from #133.

<!-- What did you change? -->

<!-- Do you want support (syntax, design, etc)? -->

<!-- Any reference material worth sharing? -->

cockerilltim committed:
Create access-comp-data

New section that will go into DesignSafe Essentials. This captures the Initial Onboarding and Requesting Allocations that were added after the TAPISv3 transition later in December, and I'm finally getting it into the documentation.

cockerilltim committed:
Update mkdocs.yml

Updating the Nav to include new section in DesignSafe Essentials.

wesleyboar commented on user-guide/access-comp-data:
```diff
@@ -0,0 +1,18 @@
+
+There are 2 ways to invoke the onboarding:
+1) Navigate to the Data Depot.
+2) Navigate to Tools & Apps, then into any of the apps and find one that uses HPC such as OpenSeesMP and click Get Started.
```
❌ Use [Markdown list syntax](https://www.markdownguide.org/basic-syntax/#lists-1), so the lists are properly rendered.

```suggestion

1. Navigate to the Data Depot.
2. Navigate to Tools & Apps, then into any of the apps and find one that uses HPC such as OpenSeesMP and click Get Started.

```

wesleyboar replied:
Fixed via 91ff5f4.

wesleyboar commented on user-guide/access-comp-data:
```diff
@@ -0,0 +1,18 @@
+2) Navigate to Tools & Apps, then into any of the apps and find one that uses HPC such as OpenSeesMP and click Get Started.
+You will be redirected to the Onboarding Setup page, and you will then want to click Request Access. 
+
+## Allocations to use the HPC-enabled Tools & Apps { #requestallocations }
```
🙋‍♂️ May we delimit our words in a heading ID?

```suggestion
## Allocations to use the HPC-enabled Tools & Apps { #request-allocations }
```

wesleyboar replied:
Changed via 91ff5f4. Reverted in 8e0ab7a8, because it was opinionated.

wesleyboar commented on user-guide/access-comp-data:
```diff
@@ -0,0 +1,18 @@
+
+
+
+
```
😶 Just one new line is adequate.

```suggestion
```

wesleyboar replied:
Changed via 91ff5f4.

wesleyboar reviewed (changes requested):
One bug. Two suggestions.

wesleyboar committed:
Merge branch 'cockerilltim-patch-2' into cockerilltim-patch-1

wesleyboar committed:
fix: doc lacks file ext and in wrong dir

wesleyboar committed:
fix: changes from initial review

https://github.com/DesignSafe-CI/DS-User-Guide/pull/132#pullrequestreview-2628191344

wesleyboar committed:
style: remove extra spaces at end of lines

wesleyboar committed:
chore: revert opinionated change

wesleyboar committed:
Merge branch 'main' into cockerilltim-patch-1

wesleyboar commented on user-guide/mkdocs.yml:
```diff
@@ -40,6 +40,8 @@ nav:
   - DesignSafe Essentials:
     - Getting Started: index.md
     - Account Help: account-help.md
+    - Accessing Computation and Data Resources: access-comp-data.md
```
The nav is very long…

<img width="480" alt="long title" src="https://github.com/user-attachments/assets/d9021597-4188-4175-ba3b-4e800545ff74" />

May we use shorter text for the nav? E.g.

> Accessing Resources

| A | B | C |
| - | - | - |
| <img width="960" alt="long title - solution 1" src="https://github.com/user-attachments/assets/7c46dc0a-6749-4497-b511-08cae16fc3a3" /> | <img width="960" alt="long title - solution 2" src="https://github.com/user-attachments/assets/7d9bb47e-2f39-4de1-a6aa-68ad95604e9c" /> | <img width="960" alt="long title - solution 3" src="https://github.com/user-attachments/assets/3bca1c24-5ab7-473a-9190-3997d23a3b2b" /> |

cockerilltim replied:
Accessing Resources - great idea. Thank you Wes!

wesleyboar replied:
Okay, this will reduce nav link text, yet retain longer page heading.

```suggestion
    - Accessing Resources: access-comp-data.md
```

wesleyboar commented on user-guide/docs/access-comp-data.md:
```diff
@@ -0,0 +1,16 @@
+
+## Allocations to use the HPC-enabled Tools & Apps { #requestallocations }
+
+DesignSafe provides many HPC-enabled Tools & Apps that require an allocation of computing time. Upon navigating to one of these, such as [OpenSeesMP] (https://www.designsafe-ci.org/use-designsafe/tools-applications/simulation/opensees/) and clicking on Get Started you will see a banner informing you to submit a ticket requesting an allocation if you don't already have one.
```
❌ Another broken link.

```suggestion
DesignSafe provides many HPC-enabled Tools & Apps that require an allocation of computing time. Upon navigating to one of these, such as [OpenSeesMP](https://www.designsafe-ci.org/use-designsafe/tools-applications/simulation/opensees/) and clicking on "Get Started" you will see a banner informing you to submit a ticket requesting an allocation if you don't already have one.
```

wesleyboar commented on user-guide/docs/access-comp-data.md:
```diff
@@ -0,0 +1,16 @@
+
+## Initial Onboarding
+
+When you are ready to begin uploading your own data to the Data Depot or want to begin using some of the Tools & Apps, you will want to initiate the onboarding process. The initial onboarding provides you with [private areas for your data] (https://www.designsafe-ci.org/user-guide/managingdata/datadepot/) and access to Tools & Apps that are hosted on our Virtual Machines such as [JupyterHub] (htt ...
```
❌ Broken links.

<img width="480" alt="broken links" src="https://github.com/user-attachments/assets/f415770a-8a0f-4945-bdbd-dddc9dc2b744" />

```suggestion
When you are ready to begin uploading your own data to the Data Depot or want to begin using some of the Tools & Apps, you will want to initiate the onboarding process. The initial onboarding provides you with [private areas for your data](https://www.designsafe-ci.org/user-guide/managingdata/datadepot/) and access to Tools & Apps that are hosted on our Virtual Machines such as [JupyterHub](https://www.designsafe-ci.org/use-designsafe/tools-applications/analysis/jupyter/) or [Interactive VM for OpenSees](https://www.designsafe-ci.org/use-designsafe/tools-applications/simulation/opensees/). Access to HPC-enabled applications requires [an additional step](#requestallocations).
```

wesleyboar reviewed (changes requested):
Nav text is rather long. Links are broken.

wesleyboar committed:
fix(access-comp-data): broken links - round 1

wesleyboar committed:
fix(access-comp-data): broken links - round 2

wesleyboar commented on user-guide/docs/access-comp-data.md:
```diff
@@ -0,0 +1,16 @@
+
+## Initial Onboarding
+
+When you are ready to begin uploading your own data to the Data Depot or want to begin using some of the Tools & Apps, you will want to initiate the onboarding process. The initial onboarding provides you with [private areas for your data](https://www.designsafe-ci.org/user-guide/managingdata/datadepot/) and access to Tools & Apps that are hosted on our Virtual Machines such as [JupyterHub](https ...
```
❌ Use a link without a domain, so the link will work on any domain (e.g. [pre-prod](https://pprd.designsafe-ci.org/user-guide/), [next](https://designsafeci-next.tacc.utexas.edu/user-guide/))

```suggestion
When you are ready to begin uploading your own data to the Data Depot or want to begin using some of the Tools & Apps, you will want to initiate the onboarding process. The initial onboarding provides you with [private areas for your data](/user-guide/managingdata/datadepot/) and access to Tools & Apps that are hosted on our Virtual Machines such as [JupyterHub](https://www.designsafe-ci.org/use-designsafe/tools-applications/analysis/jupyter/) or [Interactive VM for OpenSees](https://www.designsafe-ci.org/use-designsafe/tools-applications/simulation/opensees/). Access to HPC-enabled applications requires [an additional step](#requestallocations).
```

wesleyboar committed:
fix(access-comp-data): link only works on prod

wesleyboar committed:
enhance: reduce length of new nav text

but keep page heading as is
